Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginViewController.swift

```diff
@@ -24,7 +43,12 @@ public struct SimplifiedLoginUIFactory {
         viewModel.onClickedContinueWithoutLogin = {
             s.dismiss(animated: true, completion: nil)
         }
-        viewModel.onClickedPrivacyPolicy = {}  // TODO: Setup webview
+        
+        viewModel.onClickedPrivacyPolicy = {
```

Comment:
WebView experience should be improved. Every time you click it opens a new web view and you need to wait for content loading. Maybe it will be enough to persist WKWebView or load content earlier or listen on estimatedProgress

Reply:
Definitely. I will add this aswell. Although I am thinking that this PR has grown a little bit too much. I will do it in next pr